.teamcity/src/main/kotlin/configurations/FlakyTestQuarantine.kt

```diff
@@ -35,7 +35,7 @@ class FlakyTestQuarantine(model: CIBuildModel, stage: Stage, os: Os, arch: Arch
         val extraParameters = functionalTestExtraParameters("FlakyTestQuarantine", os, arch, testCoverage.testJvmVersion.major.toString(), testCoverage.vendor.name)
         val parameters = (
             buildToolGradleParameters(true) +
-                listOf("-PflakyTests=only") +
+                listOf("-PflakyTests=only", "-x", ":docs:quickTest", "-x", ":docs:platformTest", "-x", ":distributions-integ-tests:quickTest", "-x", ":distributions-integ-tests:platformTest") +
```

Comment:
I see that we should exclude `:docs:docsTest`, though why do we exclude all of `quickTest`?

Reply:
I misunderstood something. Removed.

Reply:
Again, why do we exclude `:docs:platformTest` and not `:docs:docsTest`?

Reply:
Because `checkSamples` is also unnecessary: https://github.com/gradle/gradle/blob/933da51984661e2c29bc08688c7ce323a091fbff/subprojects/docs/build.gradle#L805-L807